nodes/usb_cam_node.cpp

```diff
@@ -66,6 +70,9 @@ class UsbCamNode
   std::unique_ptr<diagnostic_updater::FrequencyStatus> frequency_status_;
   double expected_freq_;
 
+  ros::Timer auto_reset_exposure_timer_;
+  bool enable_auto_reset_exposure_;
```

Comment:
Wouldn't it be better to set the initial value?

Reply:
it is set at the class constructor [here](https://github.com/MisoRobotics/usb_cam/pull/46/files#diff-e158a075e24edabba175ec0488086e8d22b900d21c29cfd3361767eca5305ac7R340)